_pages/oidc.md

```diff
@@ -36,17 +36,16 @@ sidenav:
 
 ## Getting started
 
-### Choosing an authentication method
+### Client authentication
 
-Login.gov supports two ways of authenticating clients: **private_key_jwt** and **PKCE**.
+Login.gov supports **private_key_jwt** client authentication or no authentication.
 
 - **private_key_jwt** (preferred for **web apps**)
   The client sends a [JSON Web Token][jwt], or JWT, signed with a private key (minimum length of 2048 bits) when requesting access tokens. The corresponding public key is registered with the IdP ahead of time, similar to SAML.
+- *none* with **PKCE** (for **native mobile apps**)
```

Comment:
**suggestion:** the `none` phrasing seems like it could be similarly confusing:
```suggestion
- **PKCE** (preferred for **native mobile apps**)
```